packages/manager/modules/telecom-dashboard/src/identity-check/form/telecom-dashboard-identity-check-form.html

```diff
@@ -0,0 +1,178 @@
+                    data-ng-click="$ctrl.confirmCancelProcedure()"
+                >
+                    <span
+                        data-translate="telecom_dashboard_identity_check_form_cancel_procedure"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-arrow-right"
+                        data-ng-if="!$ctrl.isCancelling"
+                    ></span>
+                    <oui-spinner
+                        data-size="s"
+                        data-ng-if="$ctrl.isCancelling"
+                    ></oui-spinner>
+                </oui-button>
+            </div>
+        </oui-message>
+        <form data-ng-if="!$ctrl.procedure" name="$ctrl.form" novalidate>
+            <p
+                class="oui-heading_4"
+                data-translate="telecom_dashboard_identity_check_form_identity"
+            ></p>
+            <p
+                class="oui-heading_5"
+                data-translate="telecom_dashboard_identity_check_form_you"
+            ></p>
+            <div class="row">
+                <oui-field
+                    class="col-md-6 col-sm-12"
+                    label="{{ 'telecom_dashboard_identity_check_form_ownerLastName' | translate }}"
+                >
+                    <input
+                        class="oui-input"
+                        name="ownerLastName"
+                        placeholder="{{ 'telecom_dashboard_identity_check_form_ownerLastName' | translate }}"
+                        type="text"
+                        data-ng-model="$ctrl.model.ownerLastName"
+                        required
+                    />
+                </oui-field>
+                <oui-field
+                    class="col-md-6 col-sm-12"
+                    label="{{ 'telecom_dashboard_identity_check_form_ownerFirstName' | translate }}"
+                >
+                    <input
+                        class="oui-input"
+                        name="ownerFirstName"
+                        placeholder="{{ 'telecom_dashboard_identity_check_form_ownerFirstName' | translate }}"
+                        type="text"
+                        data-ng-model="$ctrl.model.ownerFirstName"
+                        required
+                    />
+                </oui-field>
+            </div>
+            <oui-field
+                label="{{ 'telecom_dashboard_identity_check_form_ownerAddress' | translate }}"
+            >
+                <input
+                    class="oui-input"
+                    name="ownerAddress"
+                    placeholder="{{ 'telecom_dashboard_identity_check_form_ownerAddress' | translate }}"
+                    type="text"
+                    data-ng-model="$ctrl.model.ownerAddress"
+                    required
+                />
+            </oui-field>
+            <div class="row">
+                <oui-field
+                    label="{{ 'telecom_dashboard_identity_check_form_ownerZip' | translate }}"
```

Comment:
```suggestion
                    label="{{:: 'telecom_dashboard_identity_check_form_ownerZip' | translate }}"
```
Same in other places